aws-datastore/src/test/java/com/amplifyframework/datastore/storage/sqlite/SqlCommandTest.java

```diff
@@ -489,4 +493,14 @@ private static ModelSchema getPersonModelSchemaWithCompositePrimaryKeyWithMultip
                 .modelType(Model.Type.USER)
                 .build();
     }
+
+    /**
+     * Set user agent to Flutter.
+     * @throws AmplifyException not expected.
+     */
+    private void setUserAgent() throws AmplifyException {
```

Comment:
nit: can make the function name more explicit to be clearer at line 380

```suggestion
    private void setUserAgentToFlutter() throws AmplifyException {
```